SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -530,7 +530,14 @@ void USpatialReceiver::HandleActorAuthority(const Worker_AuthorityChangeOp& Op)
 		{
 			if (USpatialActorChannel* ActorChannel = NetDriver->GetActorChannelByEntityId(Op.entity_id))
 			{
-				ActorChannel->ClientProcessOwnershipChange(Op.authority == WORKER_AUTHORITY_AUTHORITATIVE);
+				if (Op.authority == WORKER_AUTHORITY_AUTHORITATIVE)
+				{
+					ActorChannel->ClientProcessOwnershipChange(true);
+				}
+				else if (Op.authority == WORKER_AUTHORITY_NOT_AUTHORITATIVE)
+				{
+					ActorChannel->ClientProcessOwnershipChange(false);
+				}
```

Comment:
This feels like it can be made nicer but I can't think of a particularly easy way. 

Maybe add a conjunction condition to the outer if that 
`Op.authority != WORKER_AUTHORITY_AUTHORITY_LOSS_IMMINENT` 
then you can return to just having 
`ActorChannel->ClientProcessOwnershipChange(Op.authority == WORKER_AUTHORITY_AUTHORITATIVE);` ?

Reply:
Sure, I don't mind making the soft handover case explicit.